tracer_test.go

```diff
@@ -317,6 +319,30 @@ func TestGen128Bit(t *testing.T) {
 	assert.True(t, traceID.Low != 0)
 }
 
+func TestHighTraceIDGenerator(t *testing.T) {
+	hash := sha256.Sum256([]byte("x"))
+	id := binary.BigEndian.Uint64(hash[0:])
```

Comment:
don't see the point of these manipulations - why not just define some constant and check against it? Also, if the default is random generator, then there is a change it will produce the same constant, so the test must be checking that the generator was actually called, not just that the resulting HighID is equal to the constant.

Reply:
Good idea. Although the likelihood of the random generator coming up with the same value is effectively zero, still worth doing the ideal way. This was just where I temporarily moved the service name to hash transformation.